pkg/services/ngalert/schedule/schedule.go

```diff
@@ -243,13 +244,12 @@ func (sch *schedule) schedulePeriodic(ctx context.Context) error {
 				key      ngmodels.AlertRuleKey
```

Comment:
I think `AlertRuleKey` can be retrieved from `rule`? Should we keep it or remove it?